gcs.tf

```diff
@@ -0,0 +1,30 @@
+resource "google_storage_bucket" "notebook_bucket" {
```

Comment:
Where possible buckets should have `uniform_bucket_level_access=true`

Reply:
Good spot.  Have amended.